scripts/postinstall.ts

```diff
@@ -53,6 +53,7 @@ const wslDependencies = [
 // Dependencies that are specific to WSL and Lima VMs.
 const vmDependencies = [
   new tools.Trivy(),
+  new tools.SpinShim(),
```

Comment:
So if I understand correctly, `wasmShims` is the version for a bunch of different shims, but `SpinShim` downloads a specific one of that set, hence the mismatch in the naming?

Reply:
Yes, that it correct.

We potentially want to support all of them, but I don't want to bundle them all; I want to download them after installation, like we download the k3s airgap images into the cache. But that will have to wait for 1.14.0.